### PR TITLE
Added pacbio and saphyr views

### PR DESCRIPTION
**Pull request by stevieing**

### CODE REVIEW

File: src/views/Dashboard.vue [hunk 1]
```diff
@@ -0,0 +1,72 @@
+<template>
+    <div class="dashboard">
+      <div class="pipeline saphyr">
+         <router-link :to="{ path: '/saphyr' }" tag="a">
```
Comment (harrietc52) [inline src/views/Dashboard.vue]: Can we remove the description too?

**Comment by harrietc52** [comment]
RunMixin.js

The showRun() function is handling the redirect, but not specifing which pipeline
`this.$router.push({ path: `/run/${runId}` })`

Could be
`this.$router.push({ path: `{{ pipeline }}/run/${runId}` })`

**Comment by harrietc52** [comment]
SaphyrRun.js vue's Back button needs to append the pipeline too
`<router-link :to="{name: 'Runs'}">`

File: src/views/Dashboard.vue (hunk 1 above)
Comment (stevieing) [reply src/views/Dashboard.vue]: I'm going to leave it there for now. I would like it to be a placeholder so when we come to UAT it gives people a push in the direction of what they will want in there.

**Pull request merged**